sysdump/sysdump.go

```diff
@@ -1194,8 +1195,23 @@ func (c *Collector) submitCiliumBugtoolTasks(ctx context.Context, pods []*corev1
 				}
 			}()
 
+			// Default flags for cilium-bugtool
+			bugtoolFlags := []string{"--archiveType=gz"}
+			ciliumVersion, err := c.Client.GetCiliumVersion(ctx, p)
+			if err == nil {
+				// We want to run cilium-bugtool with this flag by default, but it was
+				// introduced after this version of cilium
+				if minimumVersion, err := semver.Parse("1.12.4"); err == nil {
+					if ciliumVersion.GT(minimumVersion) {
+						bugtoolFlags = append(bugtoolFlags, "--exclude-object-files")
+					}
+				}
```

Comment:
Let's use the `github.com/cilium/cilium/pkg/versioncheck` package here to parse  and check the Cilium version, like it is done in other places in the code base where we check Cilium version. Also, I think the `--exclude-object-files` flag will only be available starting with the upcoming Cilium 1.13 release, so we should check for a version ≥ 1.13.0 instead: 

```suggestion
				if versioncheck.MustCompile(">=1.13.0")(ciliumVersion) {
					bugtoolFlags = append(bugtoolFlags, "--exclude-object-files")
				}
```